src/components/Atoms/Input/Input.js

```diff
@@ -52,6 +53,9 @@ const Label = styled.label.attrs(({ id }) => ({
 }))`
   display: flex;
   flex-direction: column;
+  span {
```

Comment:
This way you will hide all `span`s in `label`. Maybe create a styled component `LabelText` and use the visibility in it.